site-src/blog/index.md

```diff
@@ -1,5 +1,22 @@
 # Blog
 
+## [Gateway API Graduates to Beta]
+
+<small style="position:relative; top:-10px;">
+  :octicons-calendar-24: July 13, 2022 ·
+  :octicons-clock-24: 5 min read
+</small>
+
+We are excited to announce the v0.5.0 release of Gateway API. For the first
+time, several of our most important Gateway API resources are graduating to
+beta. Additional, we are starting a new initiative to explore how Gateway API
+can be used for mesh and introducing new experimental concepts such as URL
+rewrites.
+
+[:octicons-arrow-right-24: Continue reading][Gateway API Graduates to Beta]
+
+[Gateway API Graduates to Beta]:/blog/2022/graduating-to-beta
```

Comment:
This URL doesn't match the one seen in https://github.com/kubernetes/website/blame/6d012150bf99db65f41408eea8471efc21be1034/content/en/blog/_posts/2022-07-13-gateway-api-in-beta.md#L6

(https://gateway-api.sigs.k8s.io/blog/2022/introducing-v1beta1/)

Let's make sure these match up.

Reply:
Woops, updated in both places, thanks!